internal/install/recipe_installer.go

```diff
@@ -92,29 +92,36 @@ func (i *RecipeInstaller) Install() error {
 	if i.RecipePathsProvided() {
 		// Load the recipes from the provided file names.
 		for _, n := range i.RecipePaths {
+			log.Debugln(fmt.Sprintf("Attempting to match recipePath %s.", n))
```

Comment:
`Debugf` is a good way to combine this into smaller code.  `log.Debugf("Attempting to match recipePath %s.", n)` is a little less to look at